src/Components/Web/src/DomWrapperInterop.cs

```diff
@@ -9,5 +9,5 @@ internal static class DomWrapperInterop
 
     public const string Focus = Prefix + "focus";
 
-    public const string FocusBySelector = Prefix + "focusBySelector";
+    public const string FocusOnNavigate = Prefix + "focusOnNavigate";
```

Comment:
I'm not sure I understand this change. The existing `FocusOnNavigate` component will still need to call `focusBySelector`, won't it?

Reply:
yes, `focusOnNavigate` will call `focusBySelector`